{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Network\/documentation\/Network\/NetworkError\/helpAnchor"},"metadata":{"modules":[{"relatedModules":["Foundation"],"name":"Network"}],"roleHeading":"Instance Property","platforms":[{"unavailable":false,"deprecated":false,"name":"iOS","introducedAt":"8.0","beta":false},{"unavailable":false,"deprecated":false,"name":"macOS","introducedAt":"10.10","beta":false},{"unavailable":false,"deprecated":false,"name":"tvOS","introducedAt":"9.0","beta":false},{"unavailable":false,"deprecated":false,"name":"watchOS","introducedAt":"2.0","beta":false}],"title":"helpAnchor","externalID":"s:10Foundation14LocalizedErrorPAAE10helpAnchorSSSgvp::SYNTHESIZED::s:7Network0A5ErrorO","symbolKind":"property","extendedModule":"Foundation","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"role":"symbol"},"sections":[],"variants":[{"paths":["\/documentation\/network\/networkerror\/helpanchor"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"patch":0,"minor":3,"major":0},"abstract":[{"type":"text","text":"Inherited from "},{"code":"LocalizedError.helpAnchor","type":"codeVoice"},{"type":"text","text":"."}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"platforms":["macOS"]}],"kind":"declarations"}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/Network\/documentation\/Network","doc:\/\/Network\/documentation\/Network\/NetworkError"]]},"references":{"doc://Network/documentation/Network/NetworkError/helpAnchor":{"title":"helpAnchor","type":"topic","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"url":"\/documentation\/network\/networkerror\/helpanchor","abstract":[],"role":"symbol","identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError\/helpAnchor","kind":"symbol"},"doc://Network/documentation/Network":{"url":"\/documentation\/network","role":"collection","identifier":"doc:\/\/Network\/documentation\/Network","abstract":[],"type":"topic","kind":"symbol","title":"Network"},"doc://Network/documentation/Network/NetworkError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NetworkError","kind":"identifier"}],"type":"topic","kind":"symbol","title":"NetworkError","identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError","role":"symbol","navigatorTitle":[{"text":"NetworkError","kind":"identifier"}],"abstract":[{"type":"text","text":"Errors that can occur during network operations."}],"url":"\/documentation\/network\/networkerror"}}}